doc/users/explain/index.rst

```diff
@@ -2,8 +2,9 @@
 
 .. redirect-from:: /users/explain
 
-Explanations
-------------
+
+API Documentation
```

Comment:
I think this is a confusing change as the difference between "API Reference" and "API Documentation" is very small in my reading.